fdbclient/ClientKnobs.cpp

```diff
@@ -302,7 +302,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 
 	init( ENABLE_ENCRYPTION_CPU_TIME_LOGGING,       false );
 	init( SIMULATION_EKP_TENANT_IDS_TO_DROP,         "-1" );
-	init( ENABLE_CONFIGURABLE_ENCRYPTION,           false );
+	init( ENABLE_CONFIGURABLE_ENCRYPTION,            true );
```

Comment:
So we want to make this true by default?

Reply:
From our last discussion I think we want to enable configurable encryption wherever it is supported. flipping the knob to true does that. cc @sfc-gh-ahusain

Reply:
I think for the time being, we should disable the flag to avoid any churn, once we bring all components together and get some soak, we shall enable by default.

Reply:
Also, my point was we don't want to _maybe_ buggify the knob and set it as default when time is apt :)